common/lib/xmodule/xmodule/vertical_block.py

```diff
@@ -216,3 +236,9 @@ def index_dictionary(self):
         xblock_body["content_type"] = "Sequence"
 
         return xblock_body
+
+    def set_discussion_status(self, is_enable_discussion):
+        self.discussions_enabled = is_enable_discussion
+
+    def get_discussion_status(self):
+        return self.discussions_enabled
```

Comment:
These are unnecessary. You can just set it directly.

Reply:
@xitij2000 in case someone turns on/off of a toggle of a unit then these functions are needed because functions in a seq_module will only be called when someone turns on/off a toggle of sequence or chapter.